src/rendering/custom-elements.js

```diff
@@ -41,6 +41,7 @@ function registerCustomElement(tagName, definitionFn) {
   WidgetClass.definitionFn = definitionFn;
   WidgetClass.prototype.init = CustomElementWidget.makeInit(tagName, definitionFn);
   WidgetClass.prototype.update = CustomElementWidget.makeUpdate();
+  WidgetClass.prototype.destroy = CustomElementWidget.makeDestroy();
```

Comment:
I remember you had instead of `makeUpdate`, `makeDestroy`, etc, just `makeWidgetClass` or something like that, and it was a very nice idea, we should probably bring that back. Because src/rendering/custom-elements.js shouldn't need to worry about init/update/destroy, we can make the widget class in custom-element-widget.js